src/FluentResults/Reasons/Error.cs

```diff
@@ -112,6 +124,18 @@ public Error CausedBy(IEnumerable<Error> errors)
             return this;
         }
 
+        /// <summary>
+        /// Set the root cause of the error
+        /// </summary>
+        public Error CausedBy(IEnumerable<IError> errors)
```

Comment:
Please don't add new methods. Only change the existing method with IEnumerable<Error> errors to IEnumerable<IError> errors.

Reply:
Done.